File: src/com/jiannlee22/gitdemo/bytedemo/ByteJavaDemo.java
package com.jiannlee22.gitdemo.bytedemo;

import java.util.Arrays;

public class ByteJavaDemo {
    public static void main(String[] args) {
        byte[] bytes = new byte[4];
        for (int i = 0; i < 4; i++) {
            bytes[i] = (byte) ((i + 190) & 0xFF);
        }
        for (byte aByte : bytes) {
            System.out.println((aByte & 0xff));
        }
    }
}

File: src/main/java/com/jiannlee22/gitdemo/bytedemo/ByteJavaDemo.kt
package com.jiannlee22.gitdemo.bytedemo

import kotlin.experimental.and
import kotlin.jvm.JvmStatic

object ByteJavaDemo {
	@JvmStatic
	fun main(args: Array<String>) {
		val bytes = ByteArray(4)
		for (i in 0..3) {
			bytes[i] = (i + 190 and 0xFF).toByte()
		}
		for (aByte in bytes) {
			println(aByte and 0xff.toByte())
		}
	}
}